pkg/sql/tests/system_table_test.go

```diff
@@ -159,50 +157,49 @@ func TestSystemTableLiterals(t *testing.T) {
-		{keys.StatementStatisticsTableID, systemschema.StatementStatisticsTableSchema, systemschema.StatementStatisticsTable},
-		{keys.TransactionStatisticsTableID, systemschema.TransactionStatisticsTableSchema, systemschema.TransactionStatisticsTable},
-		{keys.DatabaseRoleSettingsTableID, systemschema.DatabaseRoleSettingsTableSchema, systemschema.DatabaseRoleSettingsTable},
-		{keys.TenantUsageTableID, systemschema.TenantUsageTableSchema, systemschema.TenantUsageTable},
-		{keys.SQLInstancesTableID, systemschema.SQLInstancesTableSchema, systemschema.SQLInstancesTable},
+		{systemschema.NamespaceTableSchema, systemschema.NamespaceTable},
+		{systemschema.DescriptorTableSchema, systemschema.DescriptorTable},
+		{systemschema.UsersTableSchema, systemschema.UsersTable},
+		{systemschema.ZonesTableSchema, systemschema.ZonesTable},
+		{systemschema.LeaseTableSchema, systemschema.LeaseTable},
+		{systemschema.EventLogTableSchema, systemschema.EventLogTable},
+		{systemschema.RangeEventTableSchema, systemschema.RangeEventTable},
+		{systemschema.UITableSchema, systemschema.UITable},
+		{systemschema.JobsTableSchema, systemschema.JobsTable},
+		{systemschema.SettingsTableSchema, systemschema.SettingsTable},
+		{systemschema.DescIDSequenceSchema, systemschema.DescIDSequence},
+		{systemschema.TenantsTableSchema, systemschema.TenantsTable},
+		{systemschema.WebSessionsTableSchema, systemschema.WebSessionsTable},
+		{systemschema.TableStatisticsTableSchema, systemschema.TableStatisticsTable},
+		{systemschema.LocationsTableSchema, systemschema.LocationsTable},
+		{systemschema.RoleMembersTableSchema, systemschema.RoleMembersTable},
+		{systemschema.CommentsTableSchema, systemschema.CommentsTable},
+		{systemschema.ProtectedTimestampsMetaTableSchema, systemschema.ProtectedTimestampsMetaTable},
+		{systemschema.ProtectedTimestampsRecordsTableSchema, systemschema.ProtectedTimestampsRecordsTable},
+		{systemschema.RoleOptionsTableSchema, systemschema.RoleOptionsTable},
+		{systemschema.StatementBundleChunksTableSchema, systemschema.StatementBundleChunksTable},
+		{systemschema.StatementDiagnosticsRequestsTableSchema, systemschema.StatementDiagnosticsRequestsTable},
+		{systemschema.StatementDiagnosticsTableSchema, systemschema.StatementDiagnosticsTable},
+		{systemschema.ScheduledJobsTableSchema, systemschema.ScheduledJobsTable},
+		{systemschema.SqllivenessTableSchema, systemschema.SqllivenessTable},
+		{systemschema.MigrationsTableSchema, systemschema.MigrationsTable},
+		{systemschema.JoinTokensTableSchema, systemschema.JoinTokensTable},
+		{systemschema.StatementStatisticsTableSchema, systemschema.StatementStatisticsTable},
+		{systemschema.TransactionStatisticsTableSchema, systemschema.TransactionStatisticsTable},
+		{systemschema.DatabaseRoleSettingsTableSchema, systemschema.DatabaseRoleSettingsTable},
+		{systemschema.TenantUsageTableSchema, systemschema.TenantUsageTable},
+		{systemschema.SQLInstancesTableSchema, systemschema.SQLInstancesTable},
```

Comment:
Perhaps this could be further improved to programmatically guarantee to test all descriptors defined in `systemschema`? I'll look into this.

Reply:
Done.